Projects/2.Javascript/assets/js/initializeHeaderTitles.js

```diff
@@ -0,0 +1,36 @@
+const headerTitle = {
+  headerTitle: "SIMPLE & MODERN",
+  headerText: "WE MAKE THE WORLD BEAUTIFUL EVERYDAY",
+};
+
+const headerTitleSet = [
+  headerTitle,
+  headerTitle,
+  headerTitle,
+  headerTitle,
+  headerTitle,
+];
+
+export const renderHeaderTitles = () => {
+  const cardPrefix = ".monticello-header";
```

Comment:
for such var will be a good practice to create separated field called constants.js
and import them from these file